arduino/errors.go

```diff
@@ -698,21 +698,22 @@ func (e *TempDirCreationFailedError) ToRPCStatus() *status.Status {
 	return status.New(codes.Unavailable, e.Error())
 }
 
-// TempFileCreationFailedError is returned if a temp file could not be created
-type TempFileCreationFailedError struct {
-	Cause error
+// FileCreationFailedError is returned if a temp file could not be created
+type FileCreationFailedError struct {
```

Comment:
As this is an exported object, this will be a breaking change and so the standard procedure should be followed to communicate that to the users.

Reply:
I did not though about it, but you are right